packages/assets/scss/00-base/mixins/_ma-button.scss

```diff
@@ -108,9 +108,9 @@
       height: $fonts-small;
     }
   }
-  // small
+  // xsmall
 
-  @if ($size == "small") {
+  @if ($size == "xsmall") {

```

Comment:
I think we need to add a ma__button--xsmall class in _button.scss
```
  &--xsmall {

    @include ma-button-size("xsmall");
  }
```

Like the feedback button needs to use the xsmall class to match production
http://localhost:3000/?p=organisms-mass-feedback
https://mayflower.digital.mass.gov/patternlab/b/develop/?p=organisms-mass-feedback